From 729348a66d3ad9d80f9dd7d866ec8e912c735853 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Thu, 30 Jul 2009 17:29:49 +0200
Subject: [PATCH 02/19] Revert "fix iRiver T30 product ID"

This reverts commit 0cf96f41652b4ec5cdbbda0d58e2ce089afbf51c.

Needed to revert commit a24984bac808382951ed2fe3ee6a6700bf7e3839
---
 .../10freedesktop/10-usb-music-players.fdi         |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index 26257e7..9c3a2ff 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -1008,7 +1008,7 @@
             <append key="portable_audio_player.input_formats" type="strlist">audio/mpeg</append>
           </match>
 	  <!-- T30 UMS firmware v1.7 , S10 UMS firmware v1.05 -->
-	  <match key="@storage.originating_device:usb.product_id" int_outof="0x1019">
+	  <match key="@storage.originating_device:usb.product_id" int_outof="0x1023">
             <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
             <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
             <append key="portable_audio_player.output_formats" type="strlist">application/ogg</append>
-- 
1.6.4.4

